docs/anchoring-assets/anchor-platform/sep31/configuration.mdx

```diff
@@ -0,0 +1,368 @@
+
+The Anchor Platform provides the [SEP-38 RFQ API][sep38] to senders for this purpose and requires receive-side services to implement the simplified `GET /rate` and `GET /fee` endpoints for the Anchor Platform to use on the backend.
+
+To make this API available to clients, lets add the service URL to our Stellar Info File.
+
+<CodeExample>
+
+```toml
+# dev.stellar.toml
+DIRECT_PAYMENT_SERVER = "http://localhost:8080/sep31"
+WEB_AUTH_ENDPOINT = "http://localhost:8080/auth"
+KYC_SERVER = "http://localhost:8080/kyc"
+QUOTE_SERVER = "http://localhost:8080/sep38"
+```
+
+</CodeExample>
+
+Lets enable it in our environment too.
+
+<CodeExample>
+
+```bash
+# dev.env
+SEP38_ENABLED=true
+```
+
+</CodeExample>
+
+We also need to enable USDC to be used in this API, as well as add an off-chain asset it can be exchanged with.
+
+<CodeExample>
+
+```yaml
+# dev.assets.yaml
+assets:
+  - schema: stellar
+    code: USDC
+    issuer: GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5
+    distribution_account: GBLSAHONJRODSFTLOV225NZR4LHICH63RIFQTQN37L5CRTR2IMQ5UEK7
+    significant_decimals: 2
+    sep31_enabled: true
+    sep38_enabled: true
+    send: {}
+    sep31:
+      quotes_supported: true
+      quotes_required: false
+      sep12:
+        sender:
+          types:
+            sep31-sender:
+              description: customers sending to recipients
+        receiver:
+          types:
+            sep31-receiver:
+              description: customers receiving from senders
+    sep38:
+      exchangeable_assets:
+        - iso4217:BRL
+  - schema: iso4217
+    code: BRL
+    sep38_enabled: true
+    sep38:
+      exchangeable_assets:
+        - stellar:USDC:GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5
+      country_codes:
+        - BRA
+      significant_decimals: 2
+      buy_delivery_methods:
+        - name: PIX
+          description: Have BRL sent directly to your bank account.
+```
+
+</CodeExample>
+
+Lets test that your RFQ API is live! Following `docker compose up`:
+
+<CodeExample>
+
+```bash
+curl http://localhost:8080/sep38/info | jq
+```
+
+</CodeExample>
+
+You should get the following:
+
+<CodeExample>
+
+```json
+{
+  "assets": [
+    {
+      "asset": "stellar:USDC:GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5"
+    },
+    {
+      "asset": "iso4217:BRL",
+      "country_codes": ["BRA"],
+      "buy_delivery_methods": [
+        {
+          "name": "PIX",
+          "description": "Have BRL sent directly to your bank account."
+        }
+      ]
+    }
+  ]
+}
+```
+
+</CodeExample>
+
+## Configure Platform API Authentication
```

Comment:
It's essentially the same as "Securing Platform API" section in SEP-24. Should it be moved to the general section instead?